src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	
+	private String value;
```

Comment:
You might want to change this variable from `value` to something more specific like `_block`